ceph_deploy/install.py

```diff
@@ -37,7 +51,16 @@ def install(args):
 
     for hostname in args.host:
         LOG.debug('Detecting platform for host %s ...', hostname)
-        distro = hosts.get(hostname, username=args.username)
+        distro = hosts.get(
+            hostname,
+            username=args.username,
+            # XXX this should get removed once ceph packages are split for
+            # upstream. If default_release is True, it means that the user is
+            # trying to install on a RHEL machine and should expect to get RHEL
+            # packages. Otherwise, it will need to specify either a specific
+            # version, or repo, or a development branch.
+            use_rhceph=args.default_release,
```

Comment:
To be fair "default release" only means trying to isntall on a RHEL machine if the distro is in fact a RHEL machine.

If the user just did "ceph-deploy install" on an Ubuntu machine, they would get Giant, but the default_release would be true, just would not have an effect, as i understand it.